src/OpenAL/OpenToolkit.OpenAL/Native/ApiBase.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text;
+using OpenToolkit.Core.Loader;
+
+namespace OpenToolkit.Audio.OpenAL
+{
+    public abstract class ApiBase
+    {
+        private static Dictionary<string, IPlatformLibraryNameContainer> DllIntercepts = new Dictionary<string, IPlatformLibraryNameContainer>();
+
+        static ApiBase()
+        {
+            NativeLibrary.SetDllImportResolver(typeof(ApiBase).Assembly, ImportResolver);
+        }
+
+        protected static IntPtr ImportResolver(string libraryName, Assembly assembly, DllImportSearchPath? searchPath)
+        {
+            IntPtr libHandle = IntPtr.Zero;
+            if (DllIntercepts.TryGetValue(libraryName, out var libraryNameContainer))
+            {
+                string libName = libraryNameContainer.GetLibraryName();
+
+                // FIXME: Figure out the search path stuf
```

Comment:
I believe Fred wanted this comment gone, as well as the one in line 41, and I agree here again

Reply:
Fixed in latest